rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -279,6 +279,44 @@ def test_thegraph_balances_arbitrum_one(
     )
 
 
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x9531C059098e3d194fF87FebB587aB07B30B1306']])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0x9531C059098e3d194fF87FebB587aB07B30B1306']])
+@pytest.mark.parametrize('arbitrum_one_manager_connect_at_start', [(get_arbitrum_allthatnode(weight=ONE, owned=True),)])  # noqa: E501
+def test_thegraph_balances_vested_arbitrum_one(
+        arbitrum_one_inquirer: 'ArbitrumOneInquirer',
+        arbitrum_one_transaction_decoder: 'ArbitrumOneTransactionDecoder',
+        ethereum_inquirer: 'EthereumInquirer',
+        arbitrum_one_accounts: list[ChecksumEvmAddress],
+        arbitrum_one_manager_connect_at_start,
+        inquirer: 'Inquirer',  # pylint: disable=unused-argument
+) -> None:
+    """Check that balances of GRT currently vested are properly detected."""
+    expected_grt_balance = FVal('166666.67')
+    # decode the delegation transfer event which has the vested contract address as delegator_l2
+    get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=arbitrum_one_transaction_decoder.database,
+        tx_hash=deserialize_evm_tx_hash(
+            '0x48321bb00e5c5b67f080991864606dbc493051d20712735a579d7ae31eca3d78',
+        ),
+    )
+    wait_until_all_nodes_connected(
+        connect_at_start=arbitrum_one_manager_connect_at_start,
+        evm_inquirer=arbitrum_one_inquirer,
+    )
+    thegraph_balances_inquirer = ThegraphBalancesArbitrumOne(
+        database=arbitrum_one_transaction_decoder.database,
+        evm_inquirer=arbitrum_one_inquirer,
+    )
+    thegraph_balances = thegraph_balances_inquirer.query_balances()
+    user_balance = thegraph_balances[arbitrum_one_accounts[0]]
+    assert user_balance.assets[A_GRT_ARB.resolve_to_evm_token()] == Balance(
```

Comment:
can you have more than one address to ensure that the multicall and the logic that you have there when there are more than 1 balance to check works fine?